extensions-core/datasketches/src/main/java/org/apache/druid/query/aggregation/datasketches/tuple/ArrayOfDoublesSketchBuildBufferAggregator.java

```diff
@@ -94,7 +93,7 @@ public void aggregate(final ByteBuffer buf, final int position)
     // might might be considered, but it would increase complexity including relocate() support.
     final WritableMemory mem = WritableMemory.wrap(buf, ByteOrder.LITTLE_ENDIAN);
     final WritableMemory region = mem.writableRegion(position, maxIntermediateSize);
-    final Lock lock = stripedLock.getAt(lockIndex(position)).writeLock();
+    final Lock lock = stripedLock.getAt(StripedLockHelper.lockIndex(position)).writeLock();
     lock.lock();
     try {
       final ArrayOfDoublesUpdatableSketch sketch = ArrayOfDoublesSketches.wrapUpdatableSketch(region);
```

Comment:
If this statement can update the memory, please add a comment like // ArrayOfDoublesSketches.wrapUpdatableSketch() can update the memory, therefore it must be inside the critical section.`. Otherwise, please move the statement outside of the critical section and add a comment like `// ArrayOfDoublesSketches.wrapUpdatableSketch() cannot update the memory, therefore it can be outside the critical section.`.